blueprints/controller/files/__root__/__path__/__name__.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Controller definition for the <%= dasherizedModuleName %> controller
```

Comment:
It's really just a placeholder, the expectation would be that this would be replaced with a real description of the file